### ISSUE TITLE
AWS CodeBuild Project ARN

### ISSUE THREAD

**Issue by ghost** [hashicorp/terraform-provider-aws#5962]
_This issue was originally opened by @GuidoNebiolo as hashicorp/terraform#18928. It was migrated here as a result of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

### Terraform Version

```
Terraform v0.11.8
```

### Terraform Configuration Files

```hcl
resource "aws_codebuild_project" "codebuild_project" {
  name          = "${var.codebuild_name}"
  description   = "${var.codebuild_description}"
  badge_enabled = true
  build_timeout = "10"
  service_role  = "${aws_iam_role.codebuild_role.arn}"

  artifacts {
    type = "S3"
  }

  environment {
    compute_type = "BUILD_GENERAL1_SMALL"
    image        = "${var.codebuild_image}"
    type         = "LINUX_CONTAINER"
  }

  source {
    type                = "CODECOMMIT"
    location            = "${aws_codecommit_repository.repository.clone_url_http}"
    git_clone_depth     = 1
    report_build_status = false
  }
}

output "codebuild_arn" {
  value = "${aws_codebuild_project.codebuild_project.id}"
}
```

### Expected Behavior

output for codebuild_arn equals to arn:aws:codebuild:`region`:`account_id`:project/`codebuild_project_id`

### Actual Behavior

output for codebuild_arn is equal to `codebuild_project_id`

**Comment by bflad** [comment]
Hi @GuidoNebiolo 👋 

Was this particular resource imported into Terraform via `terraform import aws_codebuild_project. codebuild_project NAME`? It looks like if the resource is imported that way, the `id` attribute will actually remain just the name (unlike during resource creation where it will be the ARN). In this case we'll likely need to add a separate `arn` attribute that is always the ARN and document the current behavior. I can submit something real quick.

**Comment by bflad** [comment]
Pull request submitted to add stable `arn` attribute and improve documentation of `id` attribute: #5973

**Comment by GuidoNebiolo** [comment]
yes, I can confirm that the first time I create the resource I imported the resource via `terraform import`

**Comment by bflad** [comment]
This has been released in [version 1.38.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1380-september-26-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
